src/include/ipc/dai-amd.h

```diff
@@ -24,4 +24,11 @@ struct sof_ipc_dai_acp_params {
 	uint32_t fsync_rate;
 	uint32_t tdm_slots;
 } __attribute__((packed, aligned(4)));
+
+/* ACP Configuration Request - SOF_IPC_DAI_AMD_SDW_CONFIG */
+struct sof_ipc_dai_acp_sdw_params {
+	uint32_t reserved0;
+	uint32_t rate;
+	uint32_t channels;
+} __attribute__((packed, aligned(4)));
```

Comment:
number of bits?

Reply:
no bit-fields used.

Reply:
the question was how the bit width (16/24/32) is provided to firmware - here only rate and channels are provided.

Reply:
currently only supports 16 bit width.